crates/env_filter/Cargo.toml

```diff
@@ -18,11 +18,11 @@ rustdoc-args = ["--cfg", "docsrs"]
 
 [package.metadata.release]
 pre-release-replacements = [
-  {file="CHANGELOG.md", search="Unreleased", replace="{{version}}", min=1},
```

Comment:
Sorry, missed the `toml` file being autoformatted... Shall I try to revert this?

Reply:
Please do so as its unrelated to tis commit

Reply:
Done